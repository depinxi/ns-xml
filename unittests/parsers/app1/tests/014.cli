-a v "v is a positional argument" 