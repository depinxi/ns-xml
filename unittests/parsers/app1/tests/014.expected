CLI: "-a", "v", "v is a positional argument"
Value count: 2
Values: "v", "v is a positional argument"
Error count: 0
Subcommand: 
switchA=True
argumentB=
